docs/starters.yml

```diff
@@ -1231,3 +1231,21 @@
     - Bootstrap v4 support
     - Css Modules support
     - ESLint, Prettier
+- url: https://danshai.github.io/gatsbyv2-scientific-blog-machine-learning/
+  repo: https://github.com/DanShai/gatsbyv2-scientific-blog-machine-learning
+  description: Machine learning ready and scientific blog starter
+  tags:
+    - GatsbyV2
+    - Blog
+    - Katex
+    - Tensorflow
+    - Csv
```

Comment:
I don't see you using CSV?

Reply:
Oh thanx ! I forgot to add it in the last commit..! I will add it shortly

Reply:
Done ! csv added , its in here :  https://danshai.github.io/gatsbyv2-scientific-blog-machine-learning/projects/premierelg